### PR TITLE
New Port: Magnibox

### PR DESCRIPTION
**Pull request by mhoogcarspel**
# New Port for Magnibox

## Game Information
- **Title**: Magnibox
- **URL**: https://josephgribbin.itch.io/magnibox

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] JELOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/magnibox/Magnibox.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+cd $GAMEDIR
+
+if [ -f "${controlfolder}/libgl_${CFWNAME}.txt" ]; then 
+  source "${controlfolder}/libgl_${CFW_NAME}.txt"
+else
+  source "${controlfolder}/libgl_default.txt"
+fi
+
+# check if we have new engough version of PortMaster that contains xdelta3
+if [ $DEVICE_ARCH = "armhf" ]; then
```
Comment (kloptops) [inline ports/magnibox/Magnibox.sh:36]: This isnt needed, the `control.txt` will rename the `xdelta3.armhf` to `xdelta3`. :)

**Review by kloptops — CHANGES_REQUESTED** [review]

File: ports/magnibox/Magnibox.sh (hunk 1 above)
Comment (mhoogcarspel) [reply ports/magnibox/Magnibox.sh]: ahhh, that's the part I missed, great!

**Review by kloptops — DISMISSED** [review]
👍

**Review by Cebion — CHANGES_REQUESTED** [review]
GMloader games are always armhf only

**Review by kloptops — APPROVED** [review]
👍

**Review by Cebion — APPROVED** [review]

**Pull request merged**